README.md

```
@@ -9,10 +9,35 @@
+Toggle Native Tab Bar gives you complete control over your browser's real estate. Whether you prefer the classic horizontal layout or the new native vertical tabs, this extension allows you to toggle that bar on and off instantly whenever you need it. It is the perfect companion for sidebar extensions like **Tree Style Tab** or **Sidebery**.
 
-![Demo](https://github.com/irvinm/Toggle-Native-Tab-Bar/blob/main/Demo.gif)
+Functionality examples
+-----------------------------------------------
+
+Firefox now offers two types of sidebars that can be used in combination with this addon:
+*   **Traditional Sidebar:** The standard sidebar used by extensions like **Tree Style Tab** or **Sidebery**.
+*   **New Sidebar (Select):** The modern Firefox sidebar that allows switching between different sidebar views. 
+    *   *Note: In **Vertical Tabs** mode, these selections are integrated directly into the vertical tab bar.*
+
+#### Horizontal Tabs + Traditional Sidebar (TST)
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)